hnn_core/tests/test_parallel_backends.py

```diff
@@ -37,10 +37,8 @@ def test_gid_assignment():
     """Test that gids are assigned without overlap across ranks"""
 
     net = jones_2009_model(add_drives_from_params=False)
-    weights_ampa = {'L2_basket': 1.0, 'L2_pyramidal': 3.0,
-                    'L5_basket': 2.0, 'L5_pyramidal': 4.0}
-    syn_delays = {'L2_basket': 1.0, 'L2_pyramidal': 2.0,
-                  'L5_basket': 3.0, 'L5_pyramidal': 4.0}
+    weights_ampa = {'L2_basket': 1.0, 'L2_pyramidal': 2.0, 'L5_pyramidal': 3.0}
+    syn_delays = 0.1
```

Comment:
Were these variables copied from a separate test? Having multiple values for `syn_delays` was intentional due to #349, but those are checked elsewhere.

Reply:
Would you prefer distinct `syn_delays` values? Idk that it's necessary here but am happy to change it if we feel that it's better practice.